widget/ui/src/components/Select/select.styles.ts

```diff
@@ -0,0 +1,102 @@
+import * as Select from '@radix-ui/react-select';
+
+import { darkTheme, styled } from '../../theme';
+
+export const EXPANDABLE_QUOTE_TRANSITION_DURATION = 300;
+
+export const SelectTrigger = styled(Select.Trigger, {
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  width: '85px',
+  cursor: 'pointer',
+  backgroundColor: '$secondary',
+  padding: '$5 $10',
+  border: 0,
+  whiteSpace: 'nowrap',
+
+  '& svg': {
+    transition: `all ${EXPANDABLE_QUOTE_TRANSITION_DURATION}ms ease`,
+  },
+
+  variants: {
+    open: {
+      true: {
+        borderTopLeftRadius: '$sm',
+        borderTopRightRadius: '$sm',
+        '& ._typography, & svg': {
+          color: '$secondary',
+        },
+
+        '& svg': {
+          transform: 'rotate(180deg)',
+        },
+        backgroundColor: '$neutral100',
+        [`.${darkTheme} &`]: {
+          backgroundColor: '$neutral300',
+        },
+      },
+      false: {
+        borderRadius: '$md',
+        '& ._typography, & svg': {
+          color: '$background',
+          [`.${darkTheme} &`]: {
+            color: '$foreground',
+          },
+        },
+        '& svg': {
+          transform: 'rotate(0)',
+        },
+      },
+    },
+  },
+  '@xs': {
+    width: '119px',
+  },
+  '& ._typography': {
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    width: '52px',
+    '@xs': {
+      width: 'auto',
```

Comment:
I appreciate the approach of having a select component in the UI package. However, certain styles, such as width, make it more specific and challenging to use in other locations. Could we consider using props to allow the overriding of styles for these components? You can take inspiration from Material UI components that utilize props to modify underlying component styles.

Reply:
fixed